### PR TITLE
Separate project for classes that parse package URL and UserAgent

### PR DESCRIPTION
**Pull request by agr**
To be able to reuse our code in Synapse pipelines the code that parses URLs and UAs has been pulled into its own project with minimal amount of dependencies.

Once we stabilize the Synapse pipeline there would be another PR that would make a NuGet package out of it for easier consumption.

### CODE REVIEW

File: src/Stats.LogInterpretation/Stats.LogInterpretation.csproj [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (joelverhagen) [inline src/Stats.LogInterpretation/Stats.LogInterpretation.csproj:1]: Thanks!

**Review by zhhyu — APPROVED** [review]

File: src/Stats.ImportAzureCdnStatistics/Stats.ImportAzureCdnStatistics.csproj [hunk 2]
```diff
@@ -105,23 +95,25 @@
...
-      <Version>1.2.0</Version>
+      <Version>4.3.0</Version>
     </PackageReference>
     <PackageReference Include="MicroBuild.Core">
       <Version>0.3.0</Version>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup />
```
Comment (joelverhagen) [inline src/Stats.ImportAzureCdnStatistics/Stats.ImportAzureCdnStatistics.csproj:116]: Unnecessary

File: src/Stats.ImportAzureCdnStatistics/Stats.ImportAzureCdnStatistics.csproj [hunk 3]
```diff
@@ -105,23 +95,25 @@
...
   <ItemGroup>
     <PackageReference Include="Microsoft.Data.Services.Client">
       <Version>5.8.4</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Versioning">
-      <Version>4.3.0-preview1-2524</Version>
-    </PackageReference>
-    <PackageReference Include="UAParser">
-      <Version>1.2.0</Version>
+      <Version>4.3.0</Version>
```
Comment (joelverhagen) [inline src/Stats.ImportAzureCdnStatistics/Stats.ImportAzureCdnStatistics.csproj:108]: If we are changing version, can we align with Gallery?

**Review by joelverhagen — APPROVED** [review]

**Review by skofman1 — APPROVED** [review]
:shipit:

**Pull request merged**